splash/utils.py

```diff
@@ -185,3 +185,66 @@ def wrapper(self, *args, **kwargs):
             return meth(self, *args, **kwargs)
         return wrapper
     return decorator
+
+
+class PyResult(object):
```

Comment:
I'm asking because so far splash.utils was not aware of Lua (with a small exception of `get_alive`); PyResult looks specific to Lua scripting support.